packages/atlas/src/components/_video/VideoPlayer/VideoPlayer.tsx

```diff
@@ -548,60 +549,69 @@ const VideoPlayerComponent: ForwardRefRenderFunction<HTMLVideoElement, VideoPlay
       <div data-vjs-player onClick={handlePlayPause}>
         {needsManualPlay && (
-          <BigPlayButtonContainer onClick={handlePlayPause}>
-            <BigPlayButton onClick={handlePlayPause}>
-              <StyledSvgControlsPlay />
-            </BigPlayButton>
-          </BigPlayButtonContainer>
+          <>
+            <BigPlayButtonContainer onClick={handlePlayPause}>
+              <BigPlayButton onClick={handlePlayPause}>
+                <StyledSvgControlsPlay />
+              </BigPlayButton>
+            </BigPlayButtonContainer>
+          </>
         )}
         {isEmbedded && (
-          <EmbbeddedTopBarOverlay isFullScreen={isFullScreen}>
-            <Link to={absoluteRoutes.viewer.channel(channelId)}>
-              <Avatar
-                clickable
-                size={isFullScreen && !isMobile() ? 'cover' : 'default'}
-                assetUrl={channelAvatarUrl}
-                loading={isChannelAvatarLoading}
-              />
-            </Link>
-            <TitleContainer to={absoluteRoutes.viewer.video(videoId)} isFullscreen={isFullScreen}>
-              <StyledText variant="h300" as="h2">
-                {title}
-              </StyledText>
-              <StyledText variant="t100-strong" as="p" margin={{ top: 0.5 }}>
-                {channelTitle}
-              </StyledText>
-            </TitleContainer>
-            <PlayerControlButton
-              tooltipText="Share"
-              tooltipPosition="bottom-right"
-              onClick={(e) => e.stopPropagation()}
-            >
-              <StyledSvgControlsShare />
-            </PlayerControlButton>
-          </EmbbeddedTopBarOverlay>
+          <>
+            <EmbbeddedTopBarOverlay isFullScreen={isFullScreen}>
+              <Link to={absoluteRoutes.viewer.channel(channelId)}>
+                <Avatar
+                  clickable
+                  size={isFullScreen && !isMobile() ? 'cover' : 'default'}
+                  assetUrl={channelAvatarUrl}
+                  loading={isChannelAvatarLoading}
+                />
+              </Link>
+              <TitleContainer to={absoluteRoutes.viewer.video(videoId)} isFullscreen={isFullScreen}>
+                <StyledText variant="h300" as="h2">
+                  {title}
+                </StyledText>
+                <StyledText variant="t100-strong" as="p" margin={{ top: 0.5 }}>
+                  {channelTitle}
+                </StyledText>
+              </TitleContainer>
+              <PlayerControlButton
+                tooltipText="Share"
+                tooltipPosition="bottom-right"
+                onClick={(e) => e.stopPropagation()}
+              >
+                <StyledSvgControlsShare />
+              </PlayerControlButton>
+            </EmbbeddedTopBarOverlay>
+            {needsManualPlay && (
+              <StyledEmbeddedLogoLink to={absoluteRoutes.viewer.video(videoId)}>
```

Comment:
We need to open this in new tab, otherwise it will just change route inside `iframe`. The same goes for the player.